app/steps/ui/startpage/template.html

```diff
@@ -16,9 +16,9 @@ <h1 class="form-title heading-large">{{content.header}}</h1>
     <p>{{content.paragraph1 | safe}}</p>
     <p>{{content.paragraph2 | safe}}</p>
 
-    <ul class="panel panel-border-wide">
+    <div class="panel panel-border-wide">
         {{content.otherServices | safe}}
-    </ul>
+    </div>
```

Comment:
We should add a `<p>` tag around the content within the `<div>` to denote that this is a paragraph of text within the DOM.

This will also keep the markup consistent with the GOV.UK elements.